src/rgw/rgw_process.cc

```diff
@@ -48,7 +48,9 @@ auto schedule_request(Scheduler *scheduler, req_state *s, RGWOp *op)
 
   const auto client = op->dmclock_client();
   const auto cost = op->dmclock_cost();
-  ldpp_dout(op,10) << "scheduling with dmclock client=" << static_cast<int>(client)
+  ldpp_dout(op,10) << "scheduling with dmclock/"
```

Comment:
maybe "dmclock/" can be dropped? since the throttler has nothing to do with dmclock as such

Reply:
done